desc/objectives/_coils.py

```diff
@@ -1272,7 +1272,10 @@ def compute(self, field_params, constants=None):
 
         # B_ext is not pre-computed because field is not fixed
         B_ext = constants["field"].compute_magnetic_field(
-            x, source_grid=constants["field_grid"], basis="rpz", params=field_params
```

Comment:
No real change to this file, right? You just like to make the code longer to bother me?

Reply:
I blame my IDE

Reply:
but yea no change here just formatting